### PR TITLE
docs(bundling-configuration): update reference link

### PR DESCRIPTION
**Pull request by Nantawat-Poothong**
## Description

Update the reference links to the Bundling Configuration page

### CODE REVIEW

File: documents/src/pages/build-app/bundling-configuration.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <!--
 type: page
-title: Bundling Configuration
+title: Bundling Configuration Guide
 location: ./guides/bundling-configuration
 layout: default
 -->
 
-# Bundling Configuration
+# Bundling Configuration Guide
```
Comment (wattachai-lseg) [inline documents/src/pages/build-app/bundling-configuration.md]: Let's keep the title concise

```suggestion
# Bundling Configuration
```

File: documents/src/pages/build-app/bundling-configuration.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <!--
 type: page
-title: Bundling Configuration
+title: Bundling Configuration Guide
```
Comment (wattachai-lseg) [inline documents/src/pages/build-app/bundling-configuration.md]: Let's keep the title concise
```suggestion
title: Bundling Configuration
```

**Review by wattachai-lseg — CHANGES_REQUESTED** [review]
Just one thing.

File: documents/src/pages/build-app/bundling-configuration.md (hunk 1 above)
Comment (Nantawat-Poothong) [reply documents/src/pages/build-app/bundling-configuration.md]: Why not use Guide after the title like Angular Guide, React Guide and Vue Guide?
https://ui.refinitiv.com/tutorials/angular
https://ui.refinitiv.com/tutorials/react
https://ui.refinitiv.com/tutorials/vue

Comment (wattachai-lseg) [reply documents/src/pages/build-app/bundling-configuration.md]: I find Framework name as a page title e.g. "Angular" a little confusing. Not sure what this page is about. On the other hand, "Bundling Configuration" is easy to understand.

**Review by wattachai-lseg — APPROVED** [review]
LGTM

**Pull request merged**